build_tools/azure/install.sh

```diff
@@ -44,7 +44,7 @@ if [[ "$DISTRIB" == "conda" ]]; then
     if [[ "$UNAMESTR" == "Darwin" ]]; then
         if [[ "$SKLEARN_TEST_NO_OPENMP" != "true" ]]; then
             # on macOS, install an OpenMP-enabled clang/llvm from conda-forge.
-            TO_INSTALL="$TO_INSTALL conda-forge::compilers \
+            TO_INSTALL="$TO_INSTALL conda-forge::compilers'>=1.0.4' \
```

Comment:
I suspect:

```suggestion
            TO_INSTALL="$TO_INSTALL conda-forge::compilers>=1.0.4 \
```

Since there are already quotes around `TO_INSTALL`

Reply:
let's try